src/uwtools/rocoto.py

```diff
@@ -280,16 +283,16 @@ def _add_workflow_tasks(self, e: Element, config: dict) -> None:
             tag, name = self._tag_name(key)
             {STR.metatask: self._add_metatask, STR.task: self._add_task}[tag](e, subconfig, name)
 
-    def _config_validate(self, config_file: OptionalPath) -> None:
+    def _config_validate(self, config: Union[dict, YAMLConfig, OptionalPath]) -> None:
         """
         Validate the given YAML config.
 
-        :param config_file: Path to the YAML config (defaults to stdin).
+        :param config: YAMLConfig object or path to YAML file (None => read stdin).
         """
-        if not validate_yaml(
-            config_file=config_file, schema_file=resource_pathobj("rocoto.jsonschema")
-        ):
-            raise UWConfigError("YAML validation errors identified in %s" % config_file)
+        schema_file = resource_pathobj("rocoto.jsonschema")
+        ok = validate_yaml(schema_file=schema_file, config=config)
```

Comment:
Arg ordering isn't obvious here.

Reply:
I think I covered it in another comment, but that's how the arguments are (and have to be, as `schema_file` is required and `config` is optional) in the called function, so I've followed that ordering at the call site, too. Order isn't required for keyword arguments, of course, but if I'm looking at the call and looking at the function definition, I find it useful to be able to visually match them up in order.